app/assets/javascripts/models/beta/story.js

```diff
@@ -141,11 +146,37 @@ export const toggleStory = (story) => {
   };
 };
 
-const stateFor = (story) => {
-  if (story.state === "unscheduled" && typeof story.estimate === "number") return "unstarted";
-  if (story.state === "unstarted" && isUnestimatedFeature(story)) return "unscheduled";
-  if (!isFeature(story) && story.state === "unscheduled") return "unstarted";
-  return story.state;
+const isUnstartedState = (story, newAttributes) => 
+  !story._editing.estimate && newAttributes.estimate
+
+const isUnscheduledState = (story, newAttributes) => 
+  story._editing.estimate && (
+    isNull(newAttributes.estimate) || (
+      isUnscheduled(story._editing) && 
```

Comment:
You forgot to declare `const isUnscheduled = ({ state }) => state === status.UNSCHEDULED` :D

Reply:
Oh, I didn't do that because the function isUnscheduled already exists here:
https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/models/beta/story.js#L46